src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -253,7 +236,7 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
    *
    */
   const fieldConditionComponent = () => {
```

Comment:
As the name suggests, this could be on it's own component, move the render and other logic to new component to be used here.